seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/java/org/apache/seatunnel/example/flink/v2/SeaTunnelDingTalkApiExample.java

```diff
@@ -41,6 +45,17 @@ public static void main(String[] args) throws FileNotFoundException, URISyntaxEx
         Seatunnel.run(flinkCommand);
     }
 
+    public static void testSourceDingTalk() throws FileNotFoundException, URISyntaxException, CommandException {
+        String configFile = getTestConfigFile("/examples/fake_source_to_dingtalk.conf");
```

Comment:
`fake_source_to_dingtalk.conf` rename to `dingtalk_source_to_console.conf` is better.